components/scream/src/share/util/scream_common_physics_functions.hpp

```diff
@@ -129,6 +129,33 @@ struct PhysicsFunctions
   static ScalarT calculate_dz (const ScalarT& pseudo_density, const ScalarT& p_mid,
                                const ScalarT& T_mid, const ScalarT& qv);
 
+  //-----------------------------------------------------------------------------------------------//
+  // Calculate the volume mixing ratio given the wet mass mixing ratio:
+  //   X_vmr = X_mmr / (1 - X_mmr) * mol_weight_air/mol_weight_X
+  // where
+  //   X_vmr          is the volume mixing ratio X
+  //   X_mmr          is the mass mixing ratio of X
+  //   mol_weight_air is the molecular weight of dry air
+  //   mol_weight_X   is the molecular weight of X
+  //-----------------------------------------------------------------------------------------------//
+  template<typename ScalarT>
+  KOKKOS_INLINE_FUNCTION
+  static ScalarT calculate_vmr_from_mmr(const Real& gas_mol_weight, const ScalarT& mmr);
+
+  //-----------------------------------------------------------------------------------------------//
+  // Calculate the volume mixing ratio given the wet mass mixing ratio:
```

Comment:
This should read “Calculate the wet mass mixing ratio given the volume mixing ratio” (inverse of above function).